recipes/hyperts/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "HyperTS" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/DataCanvasIO/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: e5274e2398412220b532be8e5f9bd3baad51b8538e536ba3b7494aabc03bd042
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6, <3.10
+    - pip
+  run:
+    - python >=3.6, <3.10
+    - hypernets >=0.2.5.1, <0.2.6
+    - sktime >=0.8.1
+    - prophet ==1.0.1
+    - statsmodels >=0.12.1
+    - numpy >=1.16.5, <1.22
```

Comment:
Is the upper bound here required? 

Ref: https://github.com/DataCanvasIO/HyperTS/blob/90334143d6fbe966a35be57941c3e8ee55e7755f/requirements.txt#L6

Reply:
It needs `sktime >=0.8.1` , during CI execution, `pip check` shows following: 

```
sktime 0.10.1 has requirement numpy<1.22,>=1.21.0, but you have numpy 1.22.3.
sktime 0.10.1 has requirement scipy<1.8.0, but you have scipy 1.8.0.
```
Therefore,  I set `numpy<1.22`  to meet the requirements of sktime, setting `scipy <1.8.0` is the same reason

Ref:
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=474371&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd

Reply:
Ahh ok. Can you please open an issue with the sktime feedstock then because that is something they need to fix: https://github.com/conda-forge/sktime-feedstock

Until then, it can stay in here though.

Reply:
Thanks for your quick reply! I have created the [issue](https://github.com/conda-forge/sktime-feedstock/issues/31) for `conda-forge/sktime-feedstock` to request that it be compatible with more numpy and scipy versions. Can I limit the versions of scipy and numpy in my project to be compatible with sktime before they are solved? In this way, let me publish the project to CONDA forge for users to use, because it has been released for some time, but it has not been pushed to CONDA forge. I will remove these version restrictions after sktime fixes the problem.

Ref:
- https://github.com/conda-forge/sktime-feedstock/issues/31
- https://pypi.org/project/hyperts/

Reply:
Yes, until this is fixed you can include it here. I would just add the link to the issue as a note in the meta.yaml for later.
You can just add text to a recipe by prepending it with a #.

Reply:
Good idea, I added the issue to the recipe to mark the unresolved problem:
https://github.com/oaksharks/staged-recipes/blob/167f8cc948db018cd4d2571a95cdfe60521793c6/recipes/hyperts/meta.yaml#L27-L30